### PR TITLE
Fixed some typos and made minor improvements

### PR DESCRIPTION
**Pull request by rmartinezduque**

### CODE REVIEW

**Review by juanmaguitar — APPROVED** [review]

**Comment by juanmaguitar** [comment]
Changes approved and merged!
Thanks @rmartinezduque

File: docs/contributing/README.md [hunk 1]
```diff
@@ -8,16 +8,16 @@ Want to contribute to the project? That's great to hear! You can find more infor
...
+As you may know, Frontity is an open source project, available and free for everyone. We believe that working in an open way and sharing knowledge has the power to change the world for the better.
 
-Here are some good reasons to contribute to open source:
+Here are some more good reasons to contribute to open source:
 
-* **Connects you with people:** an open source community usually consists of people sharing similar interests, so it will allow to connect with amazing people.
-* **Improve your skills and competences:** if you are looking to practice, there is always a task for you in open source projects. This will allow you improve your knowledge.
-* **Learn from mentors and share your thoughts:** it is an incredible way to share and get expertise, and it is really refreshing.
+* **Build new connections**: an open source community usually consists of people sharing similar interests, so contributing to a project will allow you to connect with amazing people.
+* **Improve your skills and competences**: if you are looking for practice, there is always a task for you in open source projects. This will allow you improve your knowledge.
```
Comment (mburridge) [inline docs/contributing/README.md:16]: @rmartinezduque "if you are looking to practice" was correct, though I would rephrase as "if you want to practice your skills..."

File: docs/contributing/README.md [hunk 2]
```diff
@@ -8,16 +8,16 @@ Want to contribute to the project? That's great to hear! You can find more infor
...
+* **Learn from mentors and teach others**: collaborating with others on a shared project means you'll have to ask people for help, as well as share your solutions and learnings with others. This can an be a fulfilling activity for everyone involved.
 * **Get recognition**: all your work is public so you can get more visibility and recognition from others. Apart from that, you are in the perfect spot to learn about forthcoming changes.
-* **Your work matters**: you are able to make changes to a project - see how your improvements help others it is pretty gratifying.
+* **Your work matters**: you are able to make changes and improvements to a project - seing how your work helps others is pretty gratifying.
```
[the remark this answers is not in the document]
Comment (rmartinezduque) [reply docs/contributing/README.md:19]: Hi @mburridge, thanks for spotting this. I fixed a few more typos here: https://github.com/frontity/docs/pull/308. Feel free to review it and make any additional changes.
